src/Configuration/OnlyRuleResolver.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Configuration;
+
+use Rector\Contract\Rector\RectorInterface;
+use Rector\Exception\Configuration\RectorRuleNotFoundException;
+
+/**
+ * @see \Rector\Tests\Configuration\OnlyRuleResolverTest
+ */
+final class OnlyRuleResolver
+{
+    /**
+     * @param RectorInterface[] $rectors
+     */
+    public function __construct(
+        private readonly array $rectors
+    ) {
+    }
+
+    public function resolve(string $rule): string
+    {
+        $rule = ltrim($rule, '\\');
+
+        foreach ($this->rectors as $rector) {
+            if (is_a($rector, $rule, true)) {
```

Comment:
I prefer exact equal name for this case, eg: `$rector::class === $rule`, since it already ltrim-med above.

Reply:
Fixed.